data/en/bots/null.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
The name of the file needs to match the ID of your bot.

Reply:
I added it via the website, so not my fault

Reply:
Did you fill in the `Bot User ID` field?
It should have worked